index.html

```diff
@@ -26,7 +26,7 @@
 					<nav id="nav">
 						<ul>
 							<li><a href="#intro" class="active">Introduction</a></li>
-							<li><a href="#first">EOF EIPs</a></li>
+							<li><a href="#first">Why?</a></li>
```

Comment:
```suggestion
							<li><a href="#first">Motivation</a></li>
```

A suggestion to change "Why?" to "Motivation".

Reply:
I think motivation is nicer, but we can keep at why too.